### PR TITLE
[aws-secret-operator] Add chart

### PR DESCRIPTION
**Pull request by cw-sakamoto**
#### Motivation

- I created a chart for https://github.com/mumoshu/aws-secret-operator
- base manifest
  -  https://github.com/mumoshu/aws-secret-operator/tree/master/deploy/advanced
- In above manifest, the metrics port is mentioned, but when I looked at the source, it didn't seem to be implemented yet, so I dropped it temporarily.

#### Checklist

- [x] Chart Version bumped

### CODE REVIEW

File: aws-secret-operator/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.registry }}/{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command:
+            - aws-secret-operator
+          resources:
+            {{- toYaml .Values.resources | nindent 12 }}
+          env:
+            - name: WATCH_NAMESPACE
+              value: {{ if .Values.scope.singleNamespace }}"{{ .Release.Namespace }}"{{ else }}""{{ end }}
```
Comment (mumoshu) [inline aws-secret-operator/templates/deployment.yaml:40]: Can we rather define `.Values.watchNamespace` and make this line `value: {{ .Values.watchNamespace | quote }}` for simplicity?

File: aws-secret-operator/templates/role.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  - services
+  - endpoints
+  - persistentvolumeclaims
+  - events
+  - configmaps
+  - secrets
```
Comment (mumoshu) [inline aws-secret-operator/templates/role.yaml:16]: I think this is due to my fault in the upstream but this should ideally contain `secrets` only as I believe that's the only resource kind that the operator needs to access in the v1/core API group.

But I defer the decision of whether cleaning it up or keep it consistent with the upstream, to you 😃

File: aws-secret-operator/templates/deployment.yaml (hunk 1 above)
Comment (cw-sakamoto) [reply aws-secret-operator/templates/deployment.yaml:40]: Your right ! Thanks for the good suggestions😊

File: aws-secret-operator/templates/role.yaml (hunk 2 above)
Comment (cw-sakamoto) [reply aws-secret-operator/templates/role.yaml]: Sorry, the content was different between `cluserrole` and `role`.

**Comment by cw-sakamoto** [comment]
Like [aws-alb-ingress-controller](https://github.com/helm/charts/tree/master/incubator/aws-alb-ingress-controller/templates), operator itself works with cluster-wide and namespace can be specified.
So, I deleted `role` and `rolebinding`.

**Pull request merged**